### PR TITLE
Add owners for SC Migration reconciler

### PR DESCRIPTION
**Pull request by clebs**
Add codeowners for the `scmigration` reconciler.

### CODE REVIEW

**Review by tobiscr — APPROVED** [review]

File: CODEOWNERS [hunk 1]
```diff
@@ -25,5 +25,8 @@
 # Runtime Monitoring Agent (rma) reconciler
 /pkg/reconciler/instances/rma/ @ebensom @AdrianPei @fhivemind @lauraanddola @life0215 @lumi017 @Balazs23 @tobiscr @varbanv
 
+# SC Migration
+/pkg/reconciler/instances/scmigration @PK85 @piotrmiskiewicz @ksputo @szwedm @voigt @wozniakjan @hanngos @tillknuesting @ralikio
```
Comment (PK85) [inline CODEOWNERS:29]: please remove ksputo

**Comment by tobiscr** [comment]
@clebs : you can acutally merge it to MAIN directly - it's not a feature

File: CODEOWNERS (hunk 1 above)
Comment (clebs) [reply CODEOWNERS]: done!

**Review by tobiscr — APPROVED** [review]

**Comment by tobiscr** [comment]
QQ: have we verified that the change won't break this change any KCP-CLI comands?

File: internal/converters/reconciliation_info.go [hunk 2]
```diff
@@ -45,5 +45,6 @@ func ConvertOperation(operation *model.OperationEntity) keb.Operation {
 		SchedulingID:  operation.SchedulingID,
 		State:         string(operation.State),
 		Updated:       operation.Updated,
+		Type:          string(operation.Type),
```
Comment (PK85) [inline internal/converters/reconciliation_info.go:48]: Btw this PR should not contain other changes then codeowners file

**Comment by clebs** [comment]
New PR was merged here: #887 887

**Pull request closed without merging**